spec/models/registration_spec.rb

```diff
@@ -1307,4 +1307,21 @@
       expect(registration.entry_fee_with_donation(1500)).to eq(Money.new(2500, "USD"))
     end
   end
+
+  describe 'registrant_id' do
+    it 'populates upon create' do
+      expect(registration.registrant_id).to eq(1)
+    end
+
+    it 'allows registrations belonging to different competitions to have the same registrant id' do
+      expect(create(:registration)).to be_valid
```

Comment:
How is this ensured to _definitely_ be a different competition? Different from what?
(Reading the title, you should probably add assumptions that the two `registrant_id`s are indeed equal, and that the competition IDs are not equal)